packages/apriltag/src/apriltag_postprocessing_node.py

```diff
@@ -12,22 +16,21 @@ class AprilPostPros(object):
     """ """
     def __init__(self):
```

Comment:
This is not DTROS-ified yet...

Reply:
DTROS-ification is not part of this batch of changes